project.toml

```diff
@@ -3,4 +3,32 @@ requires = [
     "setuptools>=42",
     "wheel"
 ]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ivy"
+version = "{__version__}"
+description = "The unified machine learning framework, enabling framework-agnostic functions, layers and libraries."
+readme = "README.rst"
+authors = [{ name = "Unify", email = "hello@unify.ai" }]
+license = "Apache-2.0"
```

Comment:
Is there is a need for the dash? In `setup.py` it uses `Apache 2.0`

Ref: https://github.com/unifyai/ivy/blob/master/setup.py#L74